src/angular/projects/spark-core-angular/src/lib/directives/inputs/sprk-selection-input/sprk-selection-input.directive.ts

```diff
@@ -1,4 +1,4 @@
-import { Directive, ElementRef, OnInit } from '@angular/core';
+import { Directive, Input, ElementRef, OnInit } from '@angular/core';
```

Comment:
I think you forgot to remove this Input import since we are no longer having this for directives.